local-cluster/src/local_cluster.rs

```diff
@@ -173,6 +173,7 @@ impl LocalCluster {
             leader_node.info.rpc.port(),
             leader_node.info.rpc_pubsub.port(),
         ));
+        leader_config.account_paths = vec![leader_ledger_path.join("accounts")];
```

Comment:
Well, I wanted to fix this sneaky behavior as well... But I punted it...: https://github.com/solana-labs/solana/blob/4b93a7c1f6859d3771d5e6a5ec1a03f763eb12b1/runtime/src/accounts_db.rs#L472-L481